netbout-web/src/main/java/com/netbout/rest/Markdown.java

```diff
@@ -96,7 +97,7 @@ public String html() {
         synchronized (Markdown.TIDY) {
             return LINK_WHITESPACE.matcher(
                 Markdown.clean(
-                    new PegDownProcessor().markdownToHtml(
+                    new PegDownProcessor(Extensions.ALL).markdownToHtml(
```

Comment:
@antonini @dmzaytsev I didn't know about that... I don't see any problems in using these "extensions"